### DISCUSSION TITLE
Welcome to gss-ntlmssp Discussions!

### DISCUSSION THREAD

**Discussion by simo5** [gssapi/gss-ntlmssp#48] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by msaravanan77** [comment]
I'm Getting following error when I'm trying to connect from the FIPS enabled RHEL machine.

Enter-PSSession : Connecting to remote server x.x.x.x failed with the following error message : acquiring creds with username only failed An invalid name was supplied SPNEGO cannot find mechanisms to negotiate For more information, see the about_Remote_Troubleshooting Help topic.
At line:1 char:1
+ Enter-PSSession -ComputerName x.x.x.x  -Credential admin -Authent ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (x.x.x.x :String) [Enter-PSSession], PSRemotingTransportException
+ FullyQualifiedErrorId : CreateRemoteRunspaceFailed
Do I need to configure anything specific for FIPS enabled OS?

**Comment by simo5** [comment]
Gssntlmssp uses old encryption algorithms that are not allowed in FIPS mode.
I do not think there is any way to use it on a FIPS mode machine without changing the code to explicitly circmvent the restrictions imposed in FIPS mode.
